packages/ui/components/ui/box.tsx

```diff
@@ -51,12 +50,6 @@ export const Box = ({
       layout={layout ?? !!layoutId}
       layoutId={layoutId}
       className={cn('flex flex-col gap-4', variants({ spacing, state, overflow }))}
-      /**
-       * Set the border radius via the style prop so it doesn't get distorted by framer-motion.
-       *
-       * @see https://www.framer.com/motion/layout-animations/##scale-correction
-       */
-      style={{ borderRadius: RESOLVED_TAILWIND_CONFIG.theme.borderRadius.lg }}
```

Comment:
Does this effect styling somehow?

Reply:
this particular export was giving me trouble as i tried to package the tailwind config, i can revert this now that it remains private